src/SourceMap.js

```diff
@@ -103,10 +103,10 @@ export default class SourceMap {
     mappings: Array<IndexedMapping<string>>,
     lineOffset?: number = 0,
     columnOffset?: number = 0
-  ): Int32Array {
+  ): Array<number> {
     // Encode all mappings into a single typed array and make one call
     // to C++ instead of one for each mapping to improve performance.
-    let mappingBuffer = new Int32Array(mappings.length * 6);
+    let mappingBuffer = [];
```

Comment:
JS array is faster than typed array/buffer? 🤯

Reply:
Apparently

Reply:
Yeah this seems around 2x faster than master. I found a way to access the typed array directly rather than treating it like a JS object. It's over 2x faster again. 😄 https://github.com/parcel-bundler/source-map/pull/60